prime-router/docs/getting-started/fhir-functions.md

```diff
@@ -1,7 +1,70 @@
 # FHIR Functions
+FHIR functions are methods that can be run on a bundle in order to extract data from it (ex. retrieve an age or telephone area code). 
```

Comment:
We document how to invoke these functions via the FHIR Path tool, but should we also have a section and talk about how else one might invoke these functions? Like from mapping files? Currently the first line of this file is:

`FHIR functions are methods that can be run on a bundle in order to extract data from it
`
That's good to know! But, I think what is missing, crucially, is the why? Why do we want to extract data from a bundle in this way? Presumably the answer lies in how we use them in our mapping files?